snmp/tests/test_profiles.py

```diff
@@ -60,6 +60,61 @@ def run_profile_check(recording_name):
     check.check(instance)
 
 
+def test_cisco_voice(aggregator):
+    run_profile_check('cisco_icm')
+    resources = ["hrSWRunPerfMem", "hrSWRunPerfCPU", "hrSWRunStatus"]
+
+    tags = ['snmp_profile:cisco_icm', 'snmp_host:test']
+    tags += common.CHECK_TAGS
+
+    for resource in resources:
+        aggregator.assert_metric('snmp.{}'.format(resource), metric_type=aggregator.GAUGE, tags=tags)
+
+    cvps = [
+        "ccvpSipIntAvgLatency1",
+        "ccvpSipIntAvgLatency2",
+        "ccvpSipIntConnectsRcv",
+        "ccvpSipIntNewCalls",
+        "ccvpSipRtActiveCalls",
+        "ccvpSipRtTotalCallLegs",
+        "ccvpLicAggMaxPortsInUse",
```

Comment:
isn't this a count type?

Reply:
Yes, I included incorrect sim data but will update it now